### PR TITLE
Add native notifications when the fallback timer is used on Windows.

### PR DESCRIPTION
**Pull request by jamesbt365**
Is this how you do it :voidvoid:

### CODE REVIEW

**Comment by jamesbt365** [comment]
image

**Comment by vedattt** [comment]
> won't go through if the user has notifications from otd off

Does the notification get logged to at least the OTD console in this case?
So long as there is *some* indicator out there, I think that's enough. User is responsible beyond that point if they've disabled OTD notifications.

**Comment by jamesbt365** [comment]
Its still logged, just the notification won't appear.

**Comment by jamesbt365** [comment]
Rebased. No clue if it still works but i'm gonna assume it does because why wouldn't it.

File: OpenTabletDriver.Desktop/Interop/Timer/WindowsTimer.cs [hunk 1]
```diff
@@ -43,7 +43,7 @@ public unsafe void Start()
                     }
                     else
                     {
-                        Log.Write("Timer", "Unsupported interval detected, will use fallback timer. Expect high CPU usage", LogLevel.Warning);
+                        Log.WriteNotify("Timer", "Unsupported interval detected, will use fallback timer. Expect high CPU usage", LogLevel.Warning);
```
Comment (InfinityGhost) [inline OpenTabletDriver.Desktop/Interop/Timer/WindowsTimer.cs:46]: What interval? This should be a bit more specific for the user to know what to change.

**Review by InfinityGhost — CHANGES_REQUESTED** [review]

File: OpenTabletDriver.Desktop/Interop/Timer/WindowsTimer.cs (hunk 1 above)
Comment (jamesbt365) [reply OpenTabletDriver.Desktop/Interop/Timer/WindowsTimer.cs]: Fixed.

**Pull request merged**